Proteomics/Modifications/Modification.cs

```diff
@@ -127,6 +129,43 @@ public static string ModLocationOnPeptideOrProtein(string _locationRestriction)
             }
         }
 
+        public static  Modification ParseProteaseModificationsFromString(string input)
```

Comment:
is it possible to use a "normal" mod annotation structure to avoid writing a second mod parser here?

Reply:
To have the modifications added to the protease.tsv file it does not work to have the modification provided in the multi-line format that is utilized in the ReadModsFromString() method that reads in the mods from PTM lists.